### DISCUSSION TITLE
Next release timeline

### DISCUSSION THREAD

**Discussion by ahmed-basyouni** [dropbox/Store#518] [General]
Hi team, 
Currently version `alpha03` uses `ExperimentalTime` this seemed to have been [dropped from store5](https://github.com/MobileNativeFoundation/Store/pull/496) around 21st of Dec. Any specific timeline to release this version? Thanks.

**Comment by matt-ramotar** [comment]
Hey! We can release alpha04. Will shoot for EOW

**Reply by ahmed-basyouni** [reply]
Thank you so much, appreciate the help

**Comment by matt-ramotar** [comment]
Hey @ahmed-basyouni Planning to release alpha04 after #516 merges

**Comment by matt-ramotar** [comment]
https://github.com/MobileNativeFoundation/Store/releases/tag/5.0.0-alpha04

**Reply by ahmed-basyouni** [reply]
Thank you 🙌
